Rakefile

```diff
@@ -318,7 +363,7 @@ def create_anonymous_user
       anonymous_login = config['login']
       anonymous_password = config['password']
 
-      if anonymous_login and anonymous_password and not ScalarmUser.find_by_login(anonymous_login)
+      if anonymous_login and anonymous_password and not ScalarmUser.where(login: anonymous_login).first
```

Comment:
Z ciekawości - czy nie jest tak, że gdyby limit: 1 pojawiło się już w where, to where zatrzyma się po znalezieniu pierwszego wystąpienia? Wydaje mi się to sensowne, ale z drugiej strony, jeżeli dobrze kojarzę, limitowanie i inne tego typu operacje w Mongo należą do pipeliningu i wykonują się sekwencyjnie tylko z drobnymi optymalizacjami. Dokumentacja zdaje się nie mówić nic na ten temat.